ui/src/app/edge/history/history.component.html

```diff
@@ -96,7 +96,7 @@
         <ion-card>
           <ion-item lines="full" color="light">
             <ion-icon slot="start" size="large" color="warning" name="information-circle-outline"></ion-icon>
-            <ion-label class="ion-text-wrap">Historische Daten sind derzeit lokal nicht verf�gbar</ion-label>
+            <ion-label class="ion-text-wrap">Historische Daten sind derzeit lokal nicht verfügbar</ion-label>
```

Comment:
Not sure why this is marked as a change... but we should use translate here

Reply:
done